docs/components/renderer.md

```diff
@@ -29,6 +29,7 @@ The `renderer` system configures a scene's
 |-------------------------|---------------------------------------------------------------------------------|---------------|
 | antialias               | Whether to perform antialiasing. If `auto`, antialiasing is disabled on mobile. | auto          |
 | colorManagement         | Whether to use a color-managed linear workflow.                                 | false         |
+| highRefreshRate         | Oculus Browser has a 72hz optional mode vs. the default 60hz                                 | false         |
```

Comment:
Description can say what the property does a bit more directly.  

`Toggles 72fps/72hz on Oculus Browser. Will default to 60fps/60hz by default`